### PR TITLE
Replace MD5 checksum with SHA256

### PR DESCRIPTION
**Pull request by mcapuccini**
## Description
Replace MD5 checksum with SHA256. Implements linked pentest recommendation.

### CODE REVIEW

File: fedn/fedn/clients/reducer/restservice.py [hunk 1]
```diff
@@ -958,10 +958,10 @@ def checksum():
 
             file_path = os.path.join(UPLOAD_FOLDER, name)
             print("trying to get {}".format(file_path))
-            from fedn.utils.checksum import md5
+            from fedn.utils.checksum import sha
```
Comment (Wrede) [inline fedn/fedn/clients/reducer/restservice.py:961]: I know there's a lot of this in fedn, but perhaps we can start to move imports to the beginning of the file?

**Review by Wrede — CHANGES_REQUESTED** [review]

File: fedn/fedn/clients/reducer/restservice.py (hunk 1 above)
Comment (Wrede) [reply fedn/fedn/clients/reducer/restservice.py:961]: that way it will also be handled by isort

Comment (mcapuccini) [reply fedn/fedn/clients/reducer/restservice.py:961]: fixed

File: fedn/fedn/common/control/package.py [hunk 2]
```diff
@@ -164,11 +164,11 @@ def validate(self, expected_checksum):
         """
         self.expected_checksum = expected_checksum
 
-        from fedn.utils.checksum import md5
+        from fedn.utils.checksum import sha
```
Comment (Wrede) [inline fedn/fedn/common/control/package.py:167]: here as well

**Review by Wrede — CHANGES_REQUESTED** [review]

File: fedn/fedn/common/control/package.py (hunk 2 above)
Comment (mcapuccini) [reply fedn/fedn/common/control/package.py]: done

**Review by Wrede — APPROVED** [review]

**Pull request merged**